analysis/export/CSVExporter/src/main/kotlin/de/maibornwolff/codecharta/exporter/csv/CSVExporter.kt

```diff
@@ -98,6 +99,13 @@ class CSVExporter : Callable<Void>, InteractiveParser {
     }
 
     override fun getDialog(): ParserDialogInterface = ParserDialog
+    override fun isUsable(inputFile: String): Boolean {
```

Comment:
I would prefer `isApplicable` and another name for the inputFile as it is not only a file path:
```suggestion
    override fun isApplicable(resourceToBeParsed: String): Boolean {
```